doc/neps/nep-0026-missing-data-summary.rst

```diff
@@ -5,7 +5,7 @@ NEP 26 — Summary of Missing Data NEPs and discussion
 :Author: Mark Wiebe <mwwiebe@gmail.com>, Nathaniel J. Smith <njs@pobox.com>
 :Status: Deferred
 :Type: Standards Track
-:Created: 2012-04-22
+:Created: 2012-04-22 (added references: 2020-04-06)
```

Comment:
```suggestion
:Created: 2012-04-22
```
For consistency with the other NEPS, I think this should be removed. There is an `:Updated:` field that may be more appropriate.